File: C/Assignment 21.9.20/Q2.c
#include <stdio.h>
//working
void Q2(int num)
{
	int evenSum = 0, oddSum = 0;
	while(num != 0)
	{
		if((num % 10) % 2)
		{
			oddSum++;
		}
		else
		{
			evenSum++;
		}
		num /= 10;
	}
	if(evenSum > oddSum)
	{
		printf("bigger");
	}
	if(evenSum < oddSum)
	{
		printf("smaller");
	}
	if(evenSum == oddSum)
	{
		printf("equals");
	}
}

void main()
{
	int num;
	printf("enter num:");
	scanf("%d", &num);
	Q2(num);
}

File: JAVA/assignment1/src/assignment1/Q2.java
//working
package assignment1;

public class Q2 {

	public static void main(String[] args) 
	{
		char c = 'A';
		for(int i = 1; i <= 26; i++)
		{
			if(c != 'A' && c != 'E' && c != 'U' && c != 'I' && c != 'O')
				System.out.print(c);
			
			c++;
		}
	}

}
